apps/home/client/setup_home_page_modules.coffee

```diff
@@ -18,7 +18,7 @@ contexts =
   popular_artists: (module) ->
     popularArtistsTemplate artists: module.context.artists
   live_auctions: (module) ->
-    auctionTemplate auction: module.context
+    auctionTemplate auctionTimeLabel: auctionTimeLabel(module.context), auctionClosesLabel: auctionClosesLabel(module.context)
```

Comment:
Innnnterestinggg. Took me a second to parse this! That may be due to my weird formatting to begin with.

Maybe better as:
```coffeescript
auctionTemplate 
   auctionTimeLabel: auctionTimeLabel(module.context)
   auctionClosesLabel: auctionClosesLabel(module.context)
```